### ISSUE TITLE
Report not hoisted packages

### ISSUE THREAD

**Issue by TrySound** [PlayMa256/node-modules-analyzer#29]
Not hoisted packages can be detected in monorepo sub packages and in nested node_modules. They are usually the result of conflicting versions. Partially can be fixed by yarn-deduplicate tool.

**Comment by PlayMa256** [comment]
Yes i agree! The way that i have done it is capable of looking into sub node_modules inside any folder on a monorepo. I'll add more test cases tomorrow to check if that holds its truth and i'll reply back!

**Comment by PlayMa256** [comment]
Test cases added, see if that handles all scenarios!

Thank you <3
